blivet/blivet.py

```diff
@@ -173,14 +152,6 @@ def reset(self, cleanup_only=False):
             # no longer in the tree
             self.bootloader.reset()
 
-        self.roots = []
-        if flags.installer_mode:
-            self.roots = find_existing_installations(self.devicetree)
-            self.dump_state("initial")
-
-        if not flags.installer_mode:
-            self.devicetree.handle_nodev_filesystems()
```

Comment:
I think the `handle_nodev_filesystems` bit should stay since it only runs when _not_ in installer mode.